keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -189,14 +225,20 @@ Once the feature is disabled, kube-apiserver will reject the pod having expanded
 DNS configuration and kubelet will create a resolver configuration excluding the
 overage.
 
+If there is a problem with an object that already has expanded DNS
+configuration, the object should be removed manually.
+
 - **What happens if we reenable the feature if it was previously rolled back?**
 
 New objects with expanded DNS configuration will be accepted by the apiserver
 and new Pods with expanded configuration will be created by the kubelet.
 
 - **Are there any tests for feature enablement/disablement?**
 
-We will add unit tests.
+Yes.
+We verified that existing pods work with the feature enabled and already created
```

Comment:
So, to be clear, if a user creates an expanded DNS search list for a pod, it gets created, the feature is disabled, and the pod is rescheduled, what happens? It sounds like not "disrupting the system" means that the pod is successfully rescheduled but with the shorter search list. Is that correct? Please document one way or the other.

Reply:
> It sounds like not "disrupting the system" means that the pod is successfully rescheduled but with the shorter search list. 

No, as we know, one pod cannot be scheduled twice.

> When this feature is disabled, objects containing podTemplate with the expanded
DNS config cannot create new pods until that podTemplate is fixed to have the
non-expanded DNS config.

Updated the answer to this.